README.md

```diff
@@ -343,6 +343,30 @@ query param:
 > ./cloud-sql-proxy --unix-socket C:\cloudsql myproject:my-region:mysql
 > ```
 
+### Configuring Service Account Impersonation
+
+The Proxy supports [service account impersonation](https://cloud.google.com/iam/docs/impersonating-service-accounts).
+This allows the Proxy to act as a different service account, which can be useful
+for granting access to resources that are not accessible to the default service
+account.
```

Comment:
let's maybe update this wording of "default service account" to "default IAM principal" as the Proxy can be started with a service account or IAM user credentials.